migrations/versions/60_add_acknowledged_column.py

```diff
@@ -0,0 +1,41 @@
+"""empty message
+
+Revision ID: 2d2309ce61f
+Revises: 50_remove_updated_details
+Create Date: 2015-06-17 11:40:51.427138
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '60_add_acknowledged_column'
+down_revision = '100_add_open_status'
```

Comment:
I had to replay my migrations on top of the existing ones to sort the multiple heads issues. Thinking the numbers are fairly redundant?